ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/mpconfigboard.h

```diff
@@ -0,0 +1,39 @@
+//Board config for Arduino Nano RP2040 Connect.
+
+// Board and hardware specific configuration
+#define MICROPY_HW_BOARD_NAME           "Arduino Nano RP2040 Connect"
+#define MICROPY_HW_FLASH_STORAGE_BYTES  (8 * 1024 * 1024)
+
+// Enable networking and sockets.
+#define MICROPY_PY_NETWORK              (1)
+#define MICROPY_PY_USOCKET              (1)
+
+// Enable USB Mass Storage with FatFS filesystem.
+#define MICROPY_HW_USB_MSC              (1)
+#define MICROPY_HW_USB_VID              (0x2341)
+#define MICROPY_HW_USB_PID              (0x015e)
+
+// UART 1 config.
+#define MICROPY_HW_UART1_TX             (8)
+#define MICROPY_HW_UART1_RX             (9)
+#define MICROPY_HW_UART1_CTS            (10)
+#define MICROPY_HW_UART1_RTS            (11)
+
+// SPI 1 config.
+#define MICROPY_HW_SPI1_SCK             (14)
+#define MICROPY_HW_SPI1_MOSI            (11)
+#define MICROPY_HW_SPI1_MISO            (8)
+
+// Bluetooth config.
+#define MICROPY_HW_BLE_UART_ID          (1)
+#define MICROPY_HW_BLE_UART_BAUDRATE    (119600)
```

Comment:
That's a pretty slow baudrate for BLE (also quite an unusual UART baudrate)... can it be increased?

Reply:
Higher baudrates don't work, I think it must be this specific baudrate or close to it (115200 works).